snekbox/nsjail.py

```diff
@@ -3,17 +3,21 @@
 import subprocess
 import sys
 import textwrap
-from subprocess import CompletedProcess
 from tempfile import NamedTemporaryFile
 from typing import Iterable
 
 from google.protobuf import text_format
 
+# noinspection PyProtectedMember
```

Comment:
out of interest, what is pycharm complaining about to make this necessary?

Reply:
The fact that `DEBUG` isn't declared in `__all__`, which I guess I understand the intention there?

Reply:
I think we should probably just add `DEBUG` to `__all__`

Reply:
Added in [893dbd7](https://github.com/python-discord/snekbox/pull/159/commits/893dbd72d6ba2d5d6dfe979795a94d63c3791b97)